### DISCUSSION TITLE
Assigning issues

### DISCUSSION THREAD

**Discussion by eccentricdevotion** [eccentricdevotion/TARDIS#408] [General]
I have a small request for people making issues, **please don't assign them to me**, all it does is fill my email inbox with unnecessary mail. Assigning people to work on issues would make sense if there was more than one person coding the plugin, but 99% of the time it's _always_ going to be me, and I get a notification for a new issue regardless of whether it is assigned to me or not :)

**Comment by eccentricdevotion** [comment]
Fixed, thanks to @Chew and her superior knowledge of the inner workings of GitHub!

**Comment by cody2418** [comment]
Where can I find the vortex manipulator permissions?

**Reply by eccentricdevotion** [reply]
On the plugin page: https://dev.bukkit.org/projects/tardisvortexmanipulator/

**Reply by eccentricdevotion** [reply]
Though this discussion is the wrong place to ask...
